packages/autocomplete-client/README.md

```diff
@@ -0,0 +1,269 @@
+
+With `keepZeroFacets` set to `false`, `comedy` disapears:
+
+genres:
+  - [x] horror (2000)
+  - [x] thriller (214)
+
+```js
+{ keepZeroFacets : true } // default: false
+```
+
+### Matching strategy
+
+`matchingStrategy` gives you the possibility to choose how Meilisearch should handle the presence of multiple query words, see [documentation](https://docs.meilisearch.com/reference/api/search.html#matching-strategy).
+
+For example, if your query is `hello world` by default Meilisearch returns documents containing either both `hello` and `world` or documents that only contain `hello`. This is the `last` strategy, where words are stripped from the right.
+The other strategy is `all`, where both `hello` and `world` **must** be present in a document for it to be returned.
+
+
+```js
+{
+  matchingStrategy: 'all' // default last
+}
+```
+
+### Request Config
+
+You can provide a custom request configuration. Available field can be [found here](https://fetch.spec.whatwg.org/#requestinit).
+
+for example, with custom headers.
+
+```ts
+{
+  requestConfig: {
+    headers: {
+      Authorization: AUTH_TOKEN
+    },
+    credentials: 'include'
+  }
+}
+```
+
+
+### Custom HTTP client
+
+You can use your own HTTP client, for example, with [`axios`](https://github.com/axios/axios).
+
+```ts
+{
+  httpClient: async (url, opts) => {
+    const response = await $axios.request({
+      url,
+      data: opts?.body,
+      headers: opts?.headers,
+      method: (opts?.method?.toLocaleUpperCase() as Method) ?? 'GET'
+    })
+    return response.data
+  }
+}
+```
+
+## More Documentation
+
+- The open-source `autocomplete` library is widely used and well documented in the [Algolia documentation](https://www.algolia.com/doc/ui-libraries/autocomplete/introduction/what-is-autocomplete/). It provides all the necessery information to set up your autocomplete experience.
+- The [Meilisearch documentation](https://docs.meilisearch.com/).
+
+## 🤖 Compatibility with Meilisearch and Autocomplete
+
+**Supported autocomplete versions**:
+
+This package only guarantees the compatibility with the [version v1.x.x of Autocomplete](https://github.com/algolia/autocomplete). It may work with older or newer versions, but these are not tested nor officially supported at this time.
+
+**API compatibility with `autocomplete`**
+Some `autocomplete` parameters are not working using the meilisearch autocomplete client.
+
+- The autocomplete [insights] parameters (https://www.algolia.com/doc/ui-libraries/autocomplete/api-reference/autocomplete-js/autocomplete/#param-insights)
+- The [autocomplete-plugin-algolia-insights](https://www.algolia.com/doc/ui-libraries/autocomplete/api-reference/autocomplete-plugin-algolia-insights/) plugin
+
+**Supported Meilisearch versions**:
+
+This package only guarantees the compatibility with the [version v1.0.0 of Meilisearch](https://github.com/meilisearch/meilisearch/releases/tag/v1.0.0).
+
+**Node / NPM versions**:
+
+- NodeJS >= 14 <= 18
```

Comment:
Seconding Bruno's question upper. If it's compatible with Node 18, why use Node 16 in the CI?